ui/app/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.component.js

```diff
@@ -239,7 +238,11 @@ export default class GasModalPageContainer extends Component {
                 properties: {
                   speed_set: speedSet,
                   gas_mode: this.state.selectedTab,
-                  gas_fees: sumHexWEIsToRenderableFiat([this.props.value, newSwapGasTotal, this.props.customTotalSupplement], 'usd', this.props.conversionRate)?.slice(1),
+                  gas_fees: sumHexWEIsToUnformattedFiat(
```

Comment:
`this.props.value` should not be included in the "gas_fees" that are calculated here